state/cloudflare/workerskv/metadata.yaml

```diff
@@ -0,0 +1,65 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: state
+name: cloudflare.workerskv
+version: v1
+status: beta
+title: "Cloudflare Workers KV"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-cloudflare-workerskv/
+capabilities:
+  - crud
+  - ttl
+authenticationProfiles:
+  - title: "Cloudflare account ID and Token"
```

Comment:
Because `cfAccountID` and `cfAPIToken` are not required if the user doesn't want Dapr to manage the worker, please also create an additional authentication profile.

1. Rename this to "Cloudflare API token" and change the description to "Authenticate with a Cloudflare API token to let Dapr manage your Worker" (no need to repeat the title)
2. Add a new authentication profile called, for example: "No authentication", with description indicating something like "Using a pre-previsioned Worker that is not managed by Dapr doesn't require authenticating with Cloudflare".